#160530258 ch-mocha-test-fetch-single-order

Johnpaul-Borree opened the pull request:
#### What does this PR do?
test endpoint GET /api/v1/orders/orderId
#### Description of Task to be completed?
- write tests to fetch single request
- write test to return not found when id is not found
#### How should this be manually tested?
run test locally using npm run test
#### What are the relevant pivotal tracker stories?
#160530258

Johnpaul-Borree committed:
[ch-#160530258] tests GET /api/v1/orders/orderId

- write test to fetch a single order
- write test to return error when order id is not found

KaiserPhemi commented on src/controller/ordersControllers.js:
```diff
@@ -1,37 +1,46 @@
 import orders from '../models/orders';
 
+export function getAllOrders (req, res) {
+	res.status(200).json({status: 'Success', Orders: orders});
```
#### Critical
Refactor code to make it more readable. Also please follow the Airbnb style guide

KaiserPhemi commented on .eslintrc.json:
```diff
@@ -0,0 +1,29 @@
+            "always"
+        ]
+    }
+}
```
#### Nitpick 
New line

Johnpaul-Borree replied:
just finished implementing this feedback

KaiserPhemi commented on test/ordersTests.js:
```diff
@@ -0,0 +1,119 @@
+				});
+		});
+	  });
+});
```
#### Nitpick
New line

Johnpaul-Borree committed:
[#160530258] implementing feedback

- Modified .eslintrc.json using airbnb style guide
- Refactor code using airbnb style
- Add new line at end of code block
- add spaces in object with curly braces

Johnpaul-Borree committed:
Merge branch 'develop' into ch-mocha-test-fetch-single-order-#160530258

Johnpaul-Borree committed:
editing due to merge conflicts
